extensions/git/package.json

```diff
@@ -2568,6 +2568,11 @@
           "default": true,
           "description": "%config.useForcePushWithLease%"
         },
+        "git.useForcePushIfIncludes": {
+          "type": "boolean",
+          "default": true,
+          "description": "%config.useForcePushIfIncludes%"
```

Comment:
Please switch to `markdownDescription`.

Reply:
I only have to change "d" to "markdownD" in "description".